Doc/library/ssl.rst

```diff
@@ -1210,6 +1212,16 @@ SSL sockets also have the following additional methods and attributes:
    .. versionchanged:: 3.9
       IPv6 address strings no longer have a trailing new line.
 
+.. method:: SSLSocket.get_verified_chain()
+
+   Returns verified verified certificate chain provided by the other
```

Comment:
Could you have the docs describe the data type returned?

Reply:
Sure, I will do it.